commercial/app/views/hosted/guardianHostedArticle.scala.html

```diff
@@ -1,176 +1,238 @@
                             </div><!--End of social media icons !-->
                         </div>
 
                         <div class="content__article-body from-content-api js-article__body">
                             <p class="intro">
                                 Zootropolis is a city like no other. But when optimistic Judy Hopps arrives, she discovers that being a bunny on a police force of big, tough animals isn’t so easy.
                             </p>
-                            <p>
+                            <p class="text">
                                 <span class="drop-cap hosted-tone-bg--zootropolis">D</span> etermined to prove herself, she jumps at the opportunity to crack a case, even if it means partnering with a fast-talking fox, Nick Wilde. Walt Disney Animation Studios’ “Zootropolis,” is a comedy-adventure directed by Tangled’s Byron Howard.
                             </p>
 
                             <!--Beginning of the Con Artist - fox !-->
                             <div class="animal con-artist">
-                                <p class="animal__stripe-odd">
-                                    <img src="@{page.customData.conArtistPic}" class="animal__stripe-character" alt="@{page.campaign.owner}">
-                                </p>
+                                <div class="animal__stripe-odd">
+                                    @fragments.image(
+                                        ImageMedia.make(Seq(ImageAsset.make(Asset(AssetType.Image, Some("image/jpeg"), Some(page.customData.conArtistPic)), 0))),
+                                        Seq("animal__stripe-character"),
+                                        GalleryMedia.inline,
+                                        "Zootropolis Con Artist"
+                                    )
+                                </div>
                                 <div class="animal__info">
                                     <div class="animal__info-poster">
                                         <img src="@{page.customData.conArtistPoster}" alt="@{page.campaign.owner}">
                                     </div>
                                     <div class="animal__info-about">
-                                        <h3 class="animal__info-about-header hosted-tone--zootropolis">The Con Artist</h3>
-                                        <span>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Ut eget velit efficitur, viverra velit vel, vehicula sapien. Duis eu ligula augue. Donec sed ligula at massa bibendum sodales. Aenean quis lorem auctor, semper sem at, interdum ligula. Mauris lacinia pharetra augue ac gravida.</span>
-                                        <a href="#" class="animal__info-about-download">Download character poster</a>
+                                        <h3 class="animal__info-about-header hosted-tone--zootropolis">Nick - The con artist</h3>
```

Comment:
This is not very BEM-like (*you heretic* ;-)). You could simplify to `animal__header` I guess.